parts/chapters/subsections/5.3/PATHS.fodt

```diff
@@ -4316,7 +4316,7 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
     <text:p text:style-name="P13694"><text:s text:c="8"/>&apos;SCHD&apos; <text:s text:c="5"/>&apos;/DISK1/NORNE/2017/SCHD-INCLUDES&apos; <text:s text:c="25"/>/</text:p>
     <text:p text:style-name="P13694">/ <text:s text:c="6"/></text:p>
     <text:p text:style-name="P13694"/>
-    <text:p text:style-name="P11188">The above example defines “GRID” and “SCHD” aliases in the RUNSPEC section than can be used in the GRID and SCHEDULE sections of the input deck. The next example shows how to use the “GRID” alias with the INCLUDE keyword in the GRID section.</text:p>
+    <text:p text:style-name="P11188">The above example defines “<text:a xlink:href="#__RefHeading___Toc38674_784232322">GRID</text:a>” and “SCHD” aliases in the <text:a xlink:href="#__RefHeading___Toc55591_1778172979">RUNSPEC</text:a> section than can be used in the <text:a xlink:href="#__RefHeading___Toc38674_784232322">GRID</text:a> and <text:a xlink:href="#__RefHeading___Toc43945_784232322">SCHEDULE</text:a> sections of the input deck. The next example shows how to use the “<text:a xlink:href="#__RefHeading___Toc38674_784232322">GRID</text:a>” alias with the <text:a xlink:href="#__RefHeading___Toc605478_3199477706">INCLUDE</text:a> keyword in the <text:a xlink:href="#__RefHeading___Toc38674_784232322">GRID</text:a> section.</text:p>
```

Comment:
The path alias "GRID" has been mistaken for a keyword. Perhaps keyword-like strings within quotes could be ignored?